### PR TITLE
chore: Gas Rebate 25

### PR DESCRIPTION
**Pull request by chrismaree**
Motivation

Gas Rebate 24.

### CODE REVIEW

File: packages/affiliates/gas-rebate/rebates/Rebate_25.json [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+    "0xdcf99ba641c98e37df8ea31188ad7e7fa5501b09": 15.6526625,
+    "0x3b271e8f1409e8e90c484d3591143bd7e1d2a177": 15.6523625,
+    "0x11efc2638184a0382bf1bb29c4047c6111f8f3c7": 15.6525125,
+    "0x7cac2ab1cf9dc459ca88a85d72ac94de2691f1e5": 15.6517625,
+    "0x82b6a4ebc3904b7aabd4e85510045eff0d43a6bd": 9.026562499999999,
+    "0x469f91dc3253165f52bb56eaa4aee838c1c7822d": 12.547575,
+    "0xacd3dc5c6f9cb967b5e1f719001e85039b0c976f": 12.903400000000001,
+    "0xbca0fd352dce62fa1b7c7f9fa8f0c1de7fef09d9": 13.613199999999999
+  },
+  "priceData": {
```
Comment (mrice32) [inline packages/affiliates/gas-rebate/rebates/Rebate_25.json:93]: Gas prices and UMA prices all look constant? Are we not doing lookups for these anymore?

**Review by mrice32 — COMMENTED** [review]
One question about how we're computing these.

Might be worth just reassessing this program in general with @smb2796 to make sure we a) want to keep doing it and b) that we're happy with the way we're computing the amounts.

One tiny nit: maybe we should label this a chore rather than a fix.

File: packages/affiliates/gas-rebate/rebates/Rebate_25.json (hunk 1 above)
Comment (mrice32) [reply packages/affiliates/gas-rebate/rebates/Rebate_25.json:93]: We don't use premium etherscan for UMA/ETH prices, though, do we? https://github.com/UMAprotocol/protocol/blob/b80ce3919116cf7cf0eebd24c6c057f32f2f4bed/packages/affiliates/gas-rebate/VoterGasRebate.js#L528-L544

Comment (chrismaree) [reply packages/affiliates/gas-rebate/rebates/Rebate_25.json:93]: it's meant to come from coingecko.  I'm not totally sure why the price of this is wrong. the hardcoded value is pretty close to what it's been for most of this month. I can spend time looking into this but not sure how useful it is given it's pretty close to the "true" price.

Comment (mrice32) [reply packages/affiliates/gas-rebate/rebates/Rebate_25.json:93]: I think we should revamp this for next time, especially with DVM 2.0 coming up!

**Pull request merged**